etc/iscsid.conf

```diff
@@ -347,6 +347,18 @@ node.session.nr_sessions = 1
 # recovery was limited to about five minutes.
 node.session.reopen_max = 0
 
+# When the iscsi targets disappear due to any reason like ports down or
+# the network issues or any other reason, existing sessions will be retried
+# by the iscsi daemon and each retry will have a 3 seconds gap by default.
```

Comment:
We use defaultTime2Wait we get from the target during login negotiation for connection time outs. For pretty much everything else we use that hard coded 3 seconds.